requirements.txt

```diff
@@ -1,4 +1,3 @@
 click
 requests
 pyyaml
-statusdb
```

Comment:
OK - I remember some discussion about that :sweat_smile:

I noticed the most recent version of setuptools will simply skip dependencies it can't install which makes this less of a problem. But do you need to call e.g. ``pip install taca`` with some flag to get your solution working?